### PR TITLE
Add initial support for polish signals

### PR DESCRIPTION
**Pull request by besentv**
The signal images are self-made creations.

Example renderings at Wrocław station and west of it:

Screenshot_20231102_003041
Screenshot_20231102_002820

### CODE REVIEW

**Comment by besentv** [comment]
Added support for semaphores and distant signals:

Łęgajny station:
Screenshot_20231102_201101
Świdnica station (has distant signals):
Screenshot_20231102_203756
South area of Warszawa (has block signals):
Screenshot_20231102_205519

**Comment by besentv** [comment]
Removed a pink outline on some signals.

**Comment by besentv** [comment]
I removed all the metadata from the SVGs

File: signals.mss [hunk 1]
```diff
@@ -623,8 +623,12 @@ Format details:
   /* DE distant semaphore signals type Vr which                                   */
   /*  - have no railway:signal:states=* tag                                       */
   /*  - OR have railway:signal:states=* tag that does neither include Vr1 nor Vr2 */
+  /*                                                                              */
+  /* PL distant semaphore signals type On/Od/Ot                                   */
```
Comment (DerDakon) [inline signals.mss:627]: you mention 3 types here, but the lines below add only 2 of them.

File: signals.mss [hunk 2]
```diff
@@ -640,22 +644,35 @@ Format details:
...
     }
 
-    /* can display Vr 1 */
+    /* DE: can display Vr 1                                          */
     /* This rule is overwritten if the signal can show Vr 2 as well. */
-    ["distant_states"=~"^(.*;)?DE-ESO:vr1(;.*)?$"] {
+    /* PL: can display Od2/Ot2  (expect clear)                       */
+    ["distant_states"=~"^(.*;)?DE-ESO:vr1(;.*)?$"],
+    ["distant_states"=~"^(.*;)?PL-PKP:od2(;.*)?$"],
+    ["distant_states"=~"^(.*;)?PL-PKP:ot2(;.*)?$"] {
```
Comment (DerDakon) [inline signals.mss:652]: You need to add a `["feature"=…` for everything in here now. Before it were only German signals, now you could have mixed tagging. See below for main signals how this is done.

File: signals.mss [hunk 3]
```diff
@@ -1325,4 +1348,167 @@ Format details:
       }
     }
   }
+
+  /*******************************************************/
+  /* PL main, combined and distant light signals */
+  /*******************************************************/
+  ["feature"="PL-PKP:s"] {
```
Comment (DerDakon) [inline signals.mss:1355]: This is nothing that is up to you, but I wonder if we should factor out the country specific signals to their own files and only `#include` them here (however that works in CartoCSS). @Nakaner?

**Review by DerDakon — CHANGES_REQUESTED** [review]

File: signals.mss (hunk 2 above)
Comment (besentv) [reply signals.mss]: should be good now

**Comment by besentv** [comment]
So, how should I move forward with this? Just move everything into a country specific file?

**Pull request closed without merging**